src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -27,18 +36,29 @@ partial class MainService
         [ConsoleCommand("parse", Category = "Base Commands", Description = "Parse a value to its possible conversions.")]
         private void OnParseCommand(string value)
         {
+            value = Base64Fixed(value);
             var parseFunctions = new Dictionary<string, Func<string, string?>>()
             {
-                { "Address to ScriptHash", AddressToScripthash },
+                { "Address to ScriptHash (big-endian)", AddressToScripthash },
```

Comment:
Can we use `Attribute` class for the function description.?

Reply:
Good idea, I'll do it.

Reply:
Already modified